app/components/ui/radio-group.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+import { CheckIcon } from "@radix-ui/react-icons"
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group"
+
+import { cn } from "~/utils"
+
+const RadioGroup = React.forwardRef<
```

Comment:
It doesn't feel like we need forwardRef for these but also no reason to hold up the commit on this. I went ahead and made #44 to look at it later.

Reply:
no idea this is shadcn generated code